### DISCUSSION TITLE
2.0.0-beta Bug? only one top and bottom layer

### DISCUSSION THREAD

**Discussion by otzelot3** [OrcaSlicer/OrcaSlicer#4680] [General]
![Screenshot 2024-03-24 134357](https://github.com/SoftFever/OrcaSlicer/assets/155815078/744c11a3-0f5a-46b8-8808-7c4ea6a781ab)
When i use "alternating additional wall" (?) then there is only one bottom- and one top-layer instead of the chosen 3 or 4.
Ver. 2.0.0 beta

**Comment by xxxWhatzitTooya** [comment]
Set ensure vertical shell thickness to "moderate" or "critical only".

**Reply by otzelot3** [reply]
Worked! Thanks a lot!
